cmd/web.go

```diff
@@ -336,11 +336,7 @@ func runWeb(ctx *cli.Context) error {
 			}
 			defer fr.Close()
 
-			ctx.Header().Set("Cache-Control", "public,max-age=86400")
-			ctx.Header().Set("Content-Disposition", fmt.Sprintf(`inline; filename="%s"`, attach.Name))
-			// Fix #312. Attachments with , in their name are not handled correctly by Google Chrome.
-			// We must put the name in " manually.
-			if err = repo.ServeData(ctx, "\""+attach.Name+"\"", fr); err != nil {
+			if err = repo.ServeData(ctx, attach.Name, fr); err != nil {
```

Comment:
No, the quote is preserved [here][here1] and [here][here2]

[here1]: https://github.com/go-gitea/gitea/pull/270/files#diff-c333e0fd3bba06cfaf6456df14c8e32dR30
[here2]: https://github.com/go-gitea/gitea/pull/270/files#diff-c333e0fd3bba06cfaf6456df14c8e32dR32